source/schema/schema-management-upgrade.txt

```diff
@@ -0,0 +1,206 @@
+
+     - - :option:`--schemaMode <mongosqld --schemaMode>` ``custom``
+       - :option:`--schemaSource <mongosqld --schemaSource>` <dbName>
+       - *Optional:* :option:`--schemaName <mongosqld --schemaName>` <schemaName>
+
+   * - Example
+     - The following example launches |bi-short| versions 2.10 and
+       earlier in Clustered-Reader mode:
+
+       .. code-block:: sh
+
+          mongosqld --sampleMode read \
+                    --sampleSource schemaDb
+
+     - The following example launches |bi-short| versions 2.11 and later
+       in Custom mode:
+
+       .. code-block:: sh
+
+             mongosqld --schemaMode custom \
+                       --schemaSource schemaDb
+
+.. include:: /includes/sampling-ref-chart-link.rst
+
+Clustered-Writer Mode to Auto Mode
+----------------------------------
+
+Use the following table to modify your |bi-short| version 2.10 or
+earlier Clustered-Writer configuration to be compatible with version 2.11
+and later: 
+
+.. list-table::
+   :header-rows: 1
+   :stub-columns: 1
+   :widths: 10 45 45
+
+   * - 
+     - Version 2.10 and Earlier
+     - Version 2.11 and Later Upgrade Path
+
+   * - Mode
+     - Clustered-Writer Mode
+     - Auto Mode
+
+   * - Behavior
+     - The |bi-short| obtains its schema by sampling, and persists
+       information about the schema to the ``--sampleSource`` database.
+     - The |bi-short| obtain its schema by sampling and persists each
+       sampled schema to the
+       :option:`--schemaSource <mongosqld --schemaSource>` database in
+       the :ref:`DRDL <schema-with-drdl-file>` format.
+
+   * - Options
+     - 
+       - ``--sampleMode write``
+
+       - ``--sampleSource`` <dbName>
+
+       .. include:: /includes/fact-v210-schema-options.rst
+
+     - - :option:`--schemaMode <mongosqld --schemaMode>` ``auto``
+       - :option:`--schemaSource <mongosqld --schemaSource>` <dbName>
+       - *Optional:* :option:`--schemaName <mongosqld --schemaName>` <schemaName>
+
+   * - Example
+     - The following example launches |bi-short| versions 2.10 and
+       earlier in Clustered-Writer mode:
+
+       .. code-block:: sh
+
+          mongosqld --sampleMode write \
+                    --sampleSource schemaDb
+
+     - The following example launches |bi-short| versions 2.11 and later
+       in Auto mode:
+
+       .. code-block:: sh
+
+             mongosqld --schemaMode auto \
+                       --schemaSource schemaDb
+
+.. include:: /includes/sampling-ref-chart-link.rst
+
+Deprecations
+------------
+
+The ``--sampleRefreshIntervalSecs`` flag and its corresponding config
+file setting ``schema.sample.refreshIntervalSecs`` were deprecated in
+favor of
+:option:`--schemaRefreshIntervalSecs <mongosqld --schemaRefreshIntervalSecs>`
+and :setting:`schema.refreshIntervalSecs`. The old command line option
+and configuration file setting will continue to work in the 2.11
+release, switching to the new command line option or configuration file
```

Comment:
We need something to join clause 1 to clause 2 in this sentence. You could replace the comma with a semicolon and add "however," or just add "but" before "switching".

Reply:
Done, thanks.